README.rst

```diff
@@ -82,6 +82,21 @@ You can also run linting with ``tox``:
     $ tox -e lint
 
 
+Releasing a new version
+------------------------
+
+``mozilla-django-oidc`` releases are hosted in `PyPI <https://pypi.python.org/pypi/mozilla-django-oidc>`_.
+Here are the steps you need to follow in order to push a new release:
+
+* Make sure that ``HISTORY.rst`` is up-to-date focusing mostly on backwards incompatible changes
+* Bump the project version and create a commit for that
+   * Run ``bumpversion <patch|minor|major>`` a tool to automate this procedure following the `semantic versioning scheme <http://semver.org/>`_
+* Create a `signed tag <https://git-scm.com/book/tr/v2/Git-Tools-Signing-Your-Work>`_ for that version
```

Comment:
The command for that would be `git tag -s -a {VERSION} -m "Bump version: {PREVIOUS TAG} {VERSION}"`. Right? Can we add that?